specs-go/config.go

```diff
@@ -25,6 +25,8 @@ type Spec struct {
 
 	// Linux is platform specific configuration for Linux based containers.
 	Linux Linux `json:"linux" platform:"linux"`
+	// Solaris is platform specific configuration for Solaris containers.
+	Solaris Solaris `json:"solaris" platform:"solaris"`
```

Comment:
I'm not very familiar with Go structures, but I'd expect both `Linux` and `Solaris` should be pointers.